.github/workflows/update-theme.yml

```diff
@@ -0,0 +1,43 @@
+name: Update pulumi-hugo
+on:
+  schedule:
+    - cron:  '*/60 * * * *'
```

Comment:
Instead of going in this direction, could we add a dispatch command to `pulumi/pulumi-hugo` that generates a PR in this repo whenever there's a commit to `master` there?

Reply:
Yes, and this was definitely my first inclination. But since these PRs would consist only of changes to go.mod and go.sum, I suspect we'd end up with lots of merge-conflict annoyance (say, when multiple PRs queue up and one of them gets merged) and the risk of deploying changes out of order. We could probably automate around that somehow, but ultimately it seems like what we want is for changes that merge at pulumi-hugo and pass integration tests to go out in sequence, and I _think_ that accomplishes that. But I don't disagree at all with that approach in principle.

Reply:
The multi-module and conflict thinking seems reasonable enough to me to just go with this for now.